shop-info-service/package.json

```diff
@@ -4,16 +4,21 @@
   "description": "Serverless webpack example using Typescript",
   "main": "handler.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest"
   },
   "dependencies": {
+    "axios": "^0.21.0",
```

Comment:
sorry, didn't find the place where you use the axios

Reply:
I wanted to use it for second task, but i didn't